File: test/fixtures/r/multiple_urls.r
library(httr)

headers = c(
  `Content-Type` = "application/x-www-form-urlencoded"
)

data = list(
  fooo = "blah"
)

res <- httr::PUT(url = "http://localhost:28139/file.txt", httr::add_headers(.headers=headers), body = data, encode = "form")

File: test/fixtures/ruby-httparty/multiple_urls.rb
require 'httparty'

url = 'http://localhost:28139/file.txt'
headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
}
body = File.read('file.txt')
res = HTTParty.put(url, headers: headers, body: body)


url = 'http://localhost:28139'
headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
}
body = {
  'different' => 'data',
  'time' => 'now'
}
res = HTTParty.post(url, headers: headers, body: body)
